src/h5web/vis-packs/core/VisBoundary.tsx

```diff
@@ -5,7 +5,7 @@ import ErrorFallback from '../../visualizer/ErrorFallback';
 import ValueLoader from '../../visualizer/ValueLoader';
 
 interface Props {
-  resetKey: unknown;
+  resetKey?: unknown;
```

Comment:
Made `resetKey` optional as the `RgbImageVis` has no dimension mapping susceptible to trigger a reset